src/host/kvpair.rs

```diff
@@ -202,6 +202,53 @@ impl<const DEPTH: usize> MongoMerkle<DEPTH> {
         }
         Ok(())
     }
+
+    fn generate_default_node(
+        &self,
+        index: u32,
+        hash: &[u8; 32],
```

Comment:
If it is default node, shall we remove the argument of hash?

Reply:
yes, generate_default_node only need index as the input. updated the code.